### ISSUE TITLE
brkraw gui fails to convert if "SetOutput" is not pressed first

### ISSUE THREAD

**Issue by gdevenyi** [BrkRaw/bruker#4]
```
$ brkraw gui
Exception in Tkinter callback
Traceback (most recent call last):
  File "/home/gdevenyi/miniforge3/lib/python3.7/tkinter/__init__.py", line 1705, in __call__
    return self.func(*args)
  File "/home/gdevenyi/miniforge3/lib/python3.7/site-packages/brkraw/ui/main_win.py", line 185, in _save_as
    self._raw.save_as(self._scan_id, self._reco_id, filename, dir=self._output)
  File "/home/gdevenyi/miniforge3/lib/python3.7/site-packages/brkraw/lib/loader.py", line 402, in save_nifti
    output_path = os.path.join(dir, '{}.{}'.format(filename, ext))
  File "/home/gdevenyi/miniforge3/lib/python3.7/posixpath.py", line 80, in join
    a = os.fspath(a)
TypeError: expected str, bytes or os.PathLike object, not NoneType
```

Either SetOutput should be grayed out, or the output directory should default to the runtime directory of brkraw

**Comment by dvm-shlee** [comment]
@gdevenyi  Thank you for posting the issue for this as well.
I've fixed this issue as well, so from 0.3.3 rc2 version, without SetOutput, it will save onto your current folder (where you execute the GUI mode)

Please let me know if the issue is not resolved after the update.

Thank you!

**Comment by gdevenyi** [comment]
Fixed.
